core/consensus/slots/src/lib.rs

```diff
@@ -66,8 +66,9 @@ pub trait SlotCompatible {
 }
 
 /// Convert an inherent error to common error.
+#[deprecated(note = "This method doesn't make sense")]
```

Comment:
I think I'm the one who wrote that, but the message isn't great :sweat_smile:

Reply:
If it is deprecated, remove it.